### ISSUE TITLE
Denon DN900R

### ISSUE THREAD

**Issue by McHauge** [bitfocus/companion-module-requests#255]
From Facebook:
Posted by Kevin Haddock.

Hey guys.

Looking to control a Denon DN900R audio recorder via a Streamdeck and companion. I found the manual that talks about up control and serial control but a little lost as to how to translate this into useable action. Really only looking to start and stop recording.

Any ideas on how to get this done or where to start?

Thanks

https://09f3b4e35b829ef5e143-525d2f2124a3483dfbd3f17c87329990.ssl.cf2.rackcdn.com/1934/documents/DN-900R-SerialCommandProtocolGuide-v1.0r.pdf?fbclid=IwAR0LyVMIU1i6-2kK0G3QBO1OmRFxO4EWTbv46NLEQ_3X4eBn954BlO6nx2M

https://09f3b4e35b829ef5e143-525d2f2124a3483dfbd3f17c87329990.ssl.cf2.rackcdn.com/1934/documents/DN-900R-UserGuide-v1.0.pdf?fbclid=IwAR0IWv7yW60y20RuLLc3VqU2sMJb1WA63zIWh3ggfrMxkb8OPvq9-Kl57cM

**Comment by mdgnys** [comment]
Seems to use Telnet for network control.

**Comment by McHauge** [comment]
> Seems to use Telnet for network control.

correct, the simple start without feedback could be made using the module for the Denon DN-500BD MKII as a reference. there is also a request for the DN 700. might be a possibility with a combined module, where the equipment can be selected.

**Comment by obcocav** [comment]
Hey guys.  Not trying to be a pain I’m just new to this.  What kind of fee would get this done soon?

Thanks,

Kevin

**Comment by istnv** [comment]
I have time. I'll look into the protocol details.

**Comment by obcocav** [comment]
Thank you!!!

**Comment by istnv** [comment]
Starting with the 500BD module is still quite a bit of work. There is a 60 to 70% difference with the USB/SD recorder serial commands.

**Comment by obcocav** [comment]
Thanks for checking into it.  Like I said all I’m concerned with is giving my volunteers an easy access record/stop record button.  Would it be possible to get that with a bounty/fee/etc?

**Comment by istnv** [comment]
Are you on the Slack channels?

**Comment by obcocav** [comment]
No sir.  To be honest not sure what that is.  Just a volunteer church AV
guy that knows how to integrate things with sound and video but never
gotten into programming.

Thanks,

Kevin

**Comment by istnv** [comment]
Here is a link to join. Lots of companion discussion. Github doesn't have private messaging anymore.
https://join.slack.com/t/bitfocusio/shared_invite/zt-bgslk9wn-M9ZWdipzXmuhDTHFRnbsQg

**Comment by istnv** [comment]
@obcocav I sent you a message on slack

**Comment by istnv** [comment]
The DN-700R and DN-500R use the same protocol. The 500 should work with an ethernet to rs232 adapter.

**Comment by istnv** [comment]
The module is ready for review.
Covers bitfocus/companion-module-requests#142 as well.

**Comment by josephdadams** [comment]
@istnv close?

**Comment by istnv** [comment]
I rather wait for a core release to see if there are any issues there.
